src/parser/contexts.h

```diff
@@ -388,9 +388,8 @@ struct NullInstrParserCtx {
     return Ok{};
   }
   Result<> makeRefIsNull(Index) { return Ok{}; }
-
+  Result<> makeRefFunc(Index, FuncIdxT) { return Ok{}; }
```

Comment:
What's the NullInstrParserCtx for?

Reply:
The first few phases of parsing only care about finding top-level module elements like globals, functions, etc, or parsing types, so they don't need to do anything interesting with the instructions. The parser contexts for those phases inherit from `NullInstrParserCtx` to avoid having to repeat this "do nothing" behavior for each of them separately.